apps/server/src/modules/board/uc/element.uc.ts

```diff
@@ -40,10 +45,21 @@ export class ElementUc extends BaseUc {
 		return element;
 	}
 
-	async deleteElement(userId: EntityId, elementId: EntityId): Promise<void> {
+	async deleteElement(userId: EntityId, elementId: EntityId, auth = ''): Promise<void> {
 		const element = await this.getElementWithWritePermission(userId, elementId);
 
 		await this.elementService.delete(element);
+
+		if (element instanceof DrawingElement) {
```

Comment:
please avoid element specific code in this function. This needs to be part of the deletion visitor, otherwise it will not be called when the element is deleted as part of a parent being deleted.